HISTORY.md

```diff
@@ -8,6 +8,7 @@
 
 ### Public API Change
 * `DB::GetDbSessionId(std::string& session_id)` is added. `session_id` stores a unique identifier that gets reset every time the DB is opened. This DB session ID should be unique among all open DB instances on all hosts, and should be unique among re-openings of the same or other DBs. This identifier is recorded in the LOG file on the line starting with "DB Session ID:".
+* A parameter `with_checksum` is added to `BackupEngine::VerifyBackup`, which is true by default. So now `BackupEngine::VerifyBackup` verifies backup files with checksum by default. When `with_checksum` is false, `BackupEngine::VerifyBackup` only compares file sizes to verify backup files.
```

Comment:
One question is, if user enable `with_checksum`, we do not need to verify the file size anymore?

Reply:
Yes, the code logic does not verify file sizes if `with_checksum` is enabled, since when the sizes mismatch, checksum verification is very likely to fail.

Reply:
If size is available, I think it's worth checking, especially if it's going to be checked on restore. It's an inexpensive check and 32-bit checksum collision is not inconceivable.